### DISCUSSION TITLE
link to make image full screen

### DISCUSSION THREAD

**Discussion by philipshirk** [quarto-dev/quarto-cli#7951] [Q&A] [answered]
### Description

With HTML output, is it possible to make figures links that, when clicked, open a full screen view of the image? If so, does it work if the "image" is really an HTML widget (plotly plot, leaflet map, etc)?

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/prerelease/1.4/lightbox.html>.

**Comment by j-madden-m** [comment]
Hi Philip, were you able to get the images full screen in the end, if they were a html widget (e.g. plotly or ggiraph) and not a standard image (png)? Cheers

**Reply by philipshirk** [reply]
Yes, once I was able to update to the newer quarto version, I was able to get the lightbox functionality working.
